### DISCUSSION TITLE
How can i make a table using cmd fuction(net use) result in the matlab

### DISCUSSION THREAD

**Discussion by worldmap54** [community/community#60349] [Programming Help] [answered]
Body

Hi, everyone 
I wanted to check my network driver status in matlab so i got my network imformation 
to using a  following code 

%%---------------------------
```
system('net use >filename.txt')
data = importdata('filename.txt') 
data([1, 3, end]) = [];

```

 If i use this function, Data variable becomes {nx1} cell type data like this
```
status     local       remote                                      nework     
OK         Z:           \\xxx.xx.x.xx\new_folder            Microsoft Windows Network
                                                                            Microsoft Windows Network
.
.
.
 ```

-----------------------------------------------------------------------------------------
I want to convert this data to {nx4} table,,, but I can't to do this....
 please help me, guys....

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Beargruug** [comment, marked as answer]
To achieve this, you can use the textscan function to extract the data from the cell array and then create a table from it. Here's the step-by-step solution:

```
% Assuming you already have the 'data' cell array containing the network information
% data = {nx1} cell array

% Initialize variables to store the data
status = cell(size(data));
local = cell(size(data));
remote = cell(size(data));
network = cell(size(data));

% Loop through each cell in the data array and extract the information
for i = 1:numel(data)
    % Use textscan to parse the line and extract the data
    C = textscan(data{i}, '%s %s %s %s', 'Delimiter', '\t');
    status{i} = C{1};
    local{i} = C{2};
    remote{i} = C{3};
    network{i} = C{4};
end

% Create a table using the extracted information
networkTable = table(status, local, remote, network);

% Display the resulting table
disp(networkTable);
```


Make sure to replace data = {nx1} cell array with the actual network data you obtained from the file. The textscan function is used to parse the lines of the cell array and extract the required information based on the specified delimiter (tab in this case).

Note: i didn't test it. Wrote it from memory.

**Comment by worldmap54** [comment]
Thank you:)
